README.md

```diff
@@ -132,12 +155,17 @@ export default app;
     </head>
     <body>
         <!-- Include your App Component -->
-        <App />
+-       <App />
++       <my-app></my-app>
 +       <my-test-component></my-test-component>
     </body>
 </html>
 ```
 
+Additionally, you may also define the tag within your svelte component instead of with `customElement.define` as so:
+
+`<svelte:options tag="my-test-component" />`
+
```

Comment:
this could be overwhelming to new users. i think you can just drop official docs link https://svelte.dev/docs#Custom_element_API for more information on custom elements with svelte.